frontend_lib/src/helper.js

```diff
@@ -155,11 +155,17 @@ export const displayFileSize = (bytes, decimals) => {
   return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i]
 }
 
-// INFO - GB - 2019-09-03 - In this function each split is one separator that we change for a comma
-export const parserStringToList = string => {
-  let list = string.split(' ').join(',').split('\n').join(',').split(';').join(',').split(',')
-  list = list.filter(notEmptyString => notEmptyString !== '')
-  return list
+export const parserStringToList = (string, separatorList) => {
+  let mainSeparator = ''
+  separatorList.forEach((separator, index) => {
+    if(index === 0){
```

Comment:
I don't think you have to use this, you can force the mainSeparator to, let's say, a comma and be fine with it